src/Downloader.ts

```diff
@@ -109,6 +104,49 @@ class Downloader {
       mobileRestApiAvailable: false
     };
 
+    if (this.webp) {
+      imageminOptions.set('image/jpeg', {
+        plugins: [
+          imageminJpegoptim({ max: 60, stripAll: true }),
+          // imageminJpegtran(),
+          imageminWebp({ quality: 75 }),
+        ]
+      })
+      imageminOptions.set('image/png', {
+        plugins: [
+          // imageminOptiPng(),
+          imageminPngquant({ speed: 3, strip: true, dithering: 0 }),
+          imageminAdvPng({ optimizationLevel: 4, iterations: 5 }),
+          imageminWebp({ quality: 75 })
+        ]
+      })
+      imageminOptions.set('image/gif', {
+        plugins: [
+          imageminGifsicle({ optimizationLevel: 3, colors: 64 }),
+          imageminWebp({ quality: 75 })
+        ]
+      })
```

Comment:
I don't have seen that before, but I don't see any value to to imageminJpegoptim` and all the traditional optims if we do the webp conversion. We should to only the webp convertion. Make a small loop over the the three mime-types and just make it working with `imageminWebp`